util/docs/macros/index.mjs

```diff
@@ -79,9 +79,9 @@ export {
 function insertMacros(data = '', moduleJson = {}, templates = {}, schemas = {}, options = {}, version = {}) {
     let match, regex
     const methods = moduleJson.methods && moduleJson.methods.filter( method => !isEvent(method) && !isSetter(method) || (isEvent(method) && isFullyDocumentedEvent(method)) || isProviderMethod(method))
-    const additionalEvents = moduleJson.methods && moduleJson.methods.filter( method => isEvent(method) && !isFullyDocumentedEvent(method) && !isProviderMethod(m))
```

Comment:
Was this `m` instead of `method` thing just an existing bug?

Reply:
yeah, not sure where it came from, but it broke stuff...